src/ansys_sphinx_theme/__init__.py

```diff
@@ -107,6 +117,106 @@ def setup_default_html_theme_options(app):
     app.config.html_theme_options.setdefault("collapse_navigation", True)
 
 
+def fix_edit_html_page_context(
+    app: Sphinx, pagename: str, templatename: str, context: dict, doctree: document
+) -> None:
+    """Add a function that Jinja can access for returning an "edit this page" link .
+
+    This function creates an "edit this page" link for any library.
+    The link points to the corresponding file on the main branch.
+
+    Parameters
+    ----------
+    app : Sphinx
+        The Sphinx application instance for rendering the documentation.
+    pagename : str
+        The name of the current page.
+    templatename : str
+        The name of the template being used.
+    context : dict
+        The context dictionary for the page.
+    doctree : document
+        The document tree for the page.
+
+    References
```

Comment:
I was the one originally suggesting this to be a reference. Being strict, references are only used for citations.

Therefore, let us convert this into a `Notes` section.